docs/itest_eclair.md

```diff
@@ -0,0 +1,52 @@
+# Running Eclair Integration Tests
+
+These tests verify LNDK's ability to interact with an Eclair node, specifically paying BOLT 12 offers generated by Eclair.
```

Comment:
Simplify this to "These tests verify LNDK's ability to pay BOLT 12 offers generated by Eclair."